addons/website_event/controllers/main.py

```diff
@@ -198,30 +179,28 @@ def _add_event(self, event_name=None, context={}, **kwargs):
             'date_end': (date_begin + timedelta(days=(1))).strftime('%Y-%m-%d'),
             'seats_available': 1000,
         }
-        event_id = Event.create(request.cr, request.uid, vals, context=context)
-        event = Event.browse(request.cr, request.uid, event_id, context=context)
+        event = request.env['event.event'].with_context(context).create(vals)
```

Comment:
Remove the context argument of the _add_event method that is not necessary, and thus remove the with_context.